content/applications/inventory_and_mrp/purchase/products/reordering.rst

```diff
@@ -1,96 +1,81 @@
+selecting :menuselection:`Products --> Products --> Create`. To allow for reordering, make sure
+that the :guilabel:`Can be Purchased` checkbox is activated and the :guilabel:`Product Type`
+dropdown is set to `Storable Product`.
 
-.. image:: reordering/reordering-storable-product.png
+.. image:: reordering/product-configured-for-reordering.png
    :align: center
-   :alt: Set the product type in Odoo
+   :alt: Configure a product for reordering in Odoo.
 
-As you are purchasing this product from a vendor, go to the product's *Purchase tab* and add a
-vendor by clicking on *Add a line*. You can add multiple vendors, but make sure to order them
-correctly, since reordering rules always use the first vendor in a list. You can add a price, but it
-isn't necessary for the reordering rule to work. In addition, you can add a minimum quantity you
-must order to benefit from that price.
+Add a reordering rule to a product
+==================================
 
-.. note::
-   If the quantity Odoo has to reorder doesn't match the minimum quantity specified, Odoo selects
-   the next vendor on your list. If you don't have another vendor on your list, the reordering rule
-   won't work. For that purpose, you can add the same vendor multiple times with different prices
-   for different quantities.
-
-.. image:: reordering/reordering-product-vendor.png
-   :align: center
-   :alt: Add vendor to a product in Odoo
-
-.. tip::
-   By default, a draft purchase order is created. However, if you have enabled *Purchase
-   Agreements*, you can *Propose a call for tenders* instead as shown in the image above. For more
-   information, see :doc:`../manage_deals/agreements`
+After properly configuring a product, a reordering rule can be added to it by navigating to the
+product page and selecting :menuselection:`Reordering Rules --> Create`.
 
-Next, make sure the correct route is selected under the Inventory tab of your product. If you
-created your product within the Purchase app, the *Buy* route is selected by default. If you are
-looking to dropship your product, select *Dropship*.
-
-.. image:: reordering/reordering-product-routes.png
+.. image:: reordering/reordering-rules-tab.png
    :align: center
-   :alt: Choose product routes in Odoo
-
-Set up your reordering rule
-===========================
+   :alt: Access reordering rules for a product from the product page in Odoo.
 
-Open your product and click on the *Reordering Rules* button.
+Once created, the reordering rule can be configured to generate purchase orders automatically.
 
-.. image:: reordering/reordering-button.png
+.. image:: reordering/reordering-rule-configuration.png
    :align: center
-   :alt: Reordering rules button on a product in Odoo
+   :alt: Configure the reordering rule in Odoo.
 
-Once you are on the product's reordering rules page, click on *Create*.
+- :guilabel:`Location` specifies where the ordered quantities should be stored once they are
```

Comment:
Would add a contextual line here that introduces the form fields list below the image.

```Suggestion
Fill in the reordering rules form for the product by defining the following fields:

- :guilabel:`Location` specifies where the ordered quantities should be stored once they are
  received and entered into stock.
```